### ISSUE TITLE
[Feature request] Threaded messaging

### ISSUE THREAD

**Issue by raphmim** [keybase/keybase-issues#3531]
Having many overlapping conversations in a channel becomes quickly unmanageable. It would be great to have a way to group conversations in separate threads without interrupting each other.

**Comment by inpharmaticist** [comment]
I second this. I believe Keybase is superior to Slack in many ways, but this is one where slack has the better feature.

**Comment by tstrijdhorst** [comment]
One of the things it's really missing migrating from Slack.

**Comment by alangrainger** [comment]
I'm really surprised by the last of this feature. I want to migrate our entire organisation across from Slack, but threads are a killer feature.

Any channel that has more than about 8 people gets impossible to manage without threads IMO. I've reached out to @malgorithms but haven't heard back.

**Comment by olofwalker** [comment]
Ditto, need threaded conversations.

**Comment by kcseb** [comment]
Throwing a +1 to this idea in the ring. #KeybaseThreadedConversations!

**Comment by marienhof** [comment]
Yes, threading is incredibly important. Please add threading 🥺❤️

**Comment by d1onys1us** [comment]
please add threading!
